lib/assets/javascripts/cartodb/old_common/header.js

```diff
@@ -475,6 +477,16 @@ cdb.admin.Header = cdb.core.View.extend({
   _onTabClick: function(e) {
     e.preventDefault();
     window.table_router.navigate(this._generateTableUrl(e), {trigger: true});
+  },
+
+
+  _doExport: function(e) {
+    this.mapTab._doExport(e);
```

Comment:
Ok, add a note then? Inconsistent use of the underscore prefix doesn't convey information about why.